src/Functions/Security/Get-RsCatalogItemRole.ps1

```diff
@@ -0,0 +1,117 @@
+        
+        .EXAMPLE
+            Get-RsCatalogItemRole -ReportServerUri 'http://localhost/reportserver_sql2012' -Identity 'jmcgee'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver_2012 using current user's credentials and then retrieves all access for user 'jmcgee'.
+
+        .EXAMPLE
+            Get-RsCatalogItemRole -ReportServerUri 'http://localhost/reportserver_sql2012' -path '/Path/Human Resources/'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver_2012 using current user's credentials and then retrieves all access on catalog items found at '/Path/Human Resources/'.
+        
+        .EXAMPLE
+            Get-RsCatalogItemRole -Credential 'CaptainAwesome' -Identity 'jmcgee' -Recurse
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver using CaptainAwesome's credentials and then retrieves all access for user 'jmcgee' recursively.
+    #>
+
+    [CmdletBinding()]
+    param
+    (
+        [Alias('UserOrGroupName')]
+        [string]
+        $Identity,
+        
+        [Alias('ItemPath')]
+        [string]
+        $Path = "/",
+
+        [switch]
+        $Recurse,
+        
+        [string]
+        $ReportServerUri,
+        
+        [Alias('ReportServerCredentials')]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+        
+        $Proxy
+    )
+    
+    Begin
+    {
+        $Proxy = New-RsWebServiceProxyHelper -BoundParameters $PSBoundParameters
+    }
+        
+    Process
+    {
+        $Items = $Proxy.ListChildren($Path, $Recurse) 
+        $InheritParent = $true
+
+        foreach($Item in $Items)
+        {
+            foreach($Policy in $Proxy.GetPolicies($Item.path, [ref]$InheritParent))
+            {
+                # Drop Users/Groups that do not mach the Filter
+                if($Identity) {
+                    if($Policy.GroupUserName -ne $Identity) {
+                        Continue
+                    }
+                }
+                
+                # Creating an object to merge data from both sources. 
+                $CatalogItemRole = New-Object �TypeName PSCustomObject
+                $CatalogItemRole | Add-Member �MemberType NoteProperty �Name Identity �Value $Policy.GroupUserName
+                $CatalogItemRole | Add-Member �MemberType NoteProperty �Name Path �Value $($Item.Path + "/" + $Item.Name)
+                $CatalogItemRole | Add-Member �MemberType NoteProperty �Name TypeName �Value $Item.TypeName
+                $CatalogItemRole | Add-Member �MemberType NoteProperty �Name Roles �Value $Policy.Roles
+
+                Write-Output $CatalogItemRole
```

Comment:
You should use a return here instead of a write-output so the result is an object that can be used in piping or redirect to different outputs